WordPress/Resources/en.lproj/Localizable.strings

```diff
@@ -9438,6 +9478,9 @@ Please install the %3$@ to use the app with this site.";
 /* Used when the response doesn't have a valid url to display */
 "unknown url" = "unknown url";
 
+/* translators: %s: the hex color value */
+"Unlabeled color. %s" = "Unlabeled color. %s";
```

Comment:
This string without reverse-DNS key [comes from Gutenberg](https://github.com/wordpress-mobile/gutenberg-mobile/blob/9a181e9abffa7c8d51305625d2614b85ad1a9f86/bundle/ios/GutenbergNativeTranslations.swift#L309).